app/Console/Commands/MakeDefaultURLsNull.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class MakeDefaultURLsNull extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:nullurl';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change "default" URLs in the Posts table to null';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:nullurl: Starting!');
+
+        // Create the progress bar
+        $bar = $this->output->createProgressBar(Post::where('url', 'default')->count());
+
+        // Get all posts with "default" for the url
+        $posts = Post::where('url', 'default')->chunkById(100, function ($posts) use ($bar) {
```

Comment:
Lil' nitpick: this `$posts` variable isn't storing anything.

Reply:
whoops that's from pre-chunk